lib/cc/engine_registry.rb

```diff
@@ -36,6 +36,7 @@ def fetch_engine_details(engine, development: false)
           [prefix, channels.fetch(engine.channel)].join,
           metadata.fetch("command", DEFAULT_COMMAND),
           metadata.fetch("description", "(No description available)"),
+          metadata.fetch("memory", nil),
         )
```

Comment:
What do you think of moving ` def default_memory_limit` here and having that be the fallback value for this fetch?